app/src/main/java/com/example/criminalintentapp/presentation/authentication/FirebaseAuthService.kt

```diff
@@ -0,0 +1,9 @@
+package com.example.criminalintentapp.presentation.authentication
```

Comment:
Usually services have their own package, so please create a `services` package and move the Firebase auth interface and its implementation into it